<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerGP</class>
 <widget class="QDialog" name="CustomerGP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>717</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>717</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>717</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>إنشاء محموعه عملاء</string>
  </property>
  <widget class="QTableWidget" name="Qtable">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>511</width>
     <height>421</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الاسم  </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الاسم EN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الحاله</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ملاحظات</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="LB_id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LB_status">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_create">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>إنشاء</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_modify">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>تعديل</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_search">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>بحث</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_search_all">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>عرض الكل</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>71</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="desc">
      <property name="text">
       <string>الأسم</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="desc_2">
      <property name="text">
       <string>الأسم EN </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>الحاله</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="desc_3">
      <property name="text">
       <string>ملاحظات</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="LE_desc">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_desc_2">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="CMB_status"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_notes">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
